main_logic/tts_client.py

```
@@ -1946,11 +1963,161 @@ async def create_connection():
+                        
+                        # 跳过数组开始/结束标记
+                        if line in ("[", "]"):
+                            continue
+                        
+                        try:
+                            event = json.loads(line)
+                            if not process_event(event):
+                                flush_audio(force=True)
+                                return
+                        except json.JSONDecodeError:
+                            # 不完整的 JSON 或格式错误，记录警告后跳过
```

Ruff (0.15.10)

[warning] Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)